docs/src/components/ReferenceTable/PropsReferenceAccordion.tsx

```diff
@@ -12,18 +12,18 @@ import { TableCode } from '../TableCode';
 import * as ReferenceTableTooltip from './ReferenceTableTooltip';
 
 function ExpandedCode(props: React.ComponentProps<'code'>) {
-  const { className = '', ...rest } = props;
+  const { className = '', ...other } = props;
   const cleaned = className
     .split(' ')
     .filter((c) => c !== 'Code')
     .join(' ');
-  return <code {...rest} className={cleaned} />;
+  return <code {...other} className={cleaned} />;
 }
 
 function ExpandedPre(props: React.ComponentProps<'pre'>) {
   return (
-    <Accordion.Scrollable gradientColor="var(--color-gray-50)">
-      <pre {...props} className="text-xs p-0 m-0" style={{ backgroundColor: 'none' }} />
+    <Accordion.Scrollable component="div" gradientColor="var(--color-gray-50)">
+      <pre {...props} className="text-xs p-0 m-0" style={{ backgroundColor: undefined }} />
```

Comment:
`background-color: none` doesn't exist: https://stackoverflow.com/questions/8739665/is-background-colornone-valid-css.